src/steps/extPaintNextSteps.js

```diff
@@ -0,0 +1,62 @@
+import ButtonSection from 'components/ButtonSection';
+import QuestionSection from 'components/QuestionSection';
+import SummarySection from 'components/SummarySection'
+import {extPaintContent} from './productContents/exteriorPaint';
+
+export default [
+  {
+    sectionName: "which",
+    hasBorderEffect: true,
+    getTitleFromLastValue: true,
+    sectionComponent: ButtonSection,
+    labels: {
+      mainTitle: "Which?",
+      subTitle: "Which kind of roof would you like to renovate in your house?"
```

Comment:
It's not the roof they want to renovate, right? maybe the first step should be the paint choosing step, and this step should have the `hasBorderEffect`